src/base-styles.css

```diff
@@ -16,23 +16,31 @@ body {
 }
 
 .iphone-notch-top-spacing {
-  padding-top: constant(safe-area-inset-top); /* iOS 11.0 */
-  padding-top: env(safe-area-inset-top); /* iOS 11.2 */
+  padding-top: constant(safe-area-inset-top);
```

Comment:
What's the reason behind those changes? Looks like that was not intended 😉

Reply:
My prettier extension moved them to a new line, don't ask me why